Doc/Tutorial/TutorialBasicMonomorphic.lhs

```diff
@@ -297,9 +288,6 @@ purpose, which is just a notational convenience.
 >   def = BirthdayFieldNullable <$> P.lmap bdNameField D.def
 >                                <*> P.lmap bdDayField  D.def
 
-Again, this is all derivable using `Generic` or Template Haskell, if
-someone would take the time to implement it.
-
```

Comment:
This comment is specifically about deriving for *monomorphic* record types, so the comment is still true.

Reply:
Reverted.